ow_faults/src/FaultInjector.cpp

```diff
@@ -12,9 +12,13 @@ constexpr std::bitset<10> FaultInjector::isPanTiltExecutionError;
 constexpr std::bitset<10> FaultInjector::isArmExecutionError;
 constexpr std::bitset<10> FaultInjector::isPowerSystemFault;
 
+constexpr std::bitset<3> FaultInjector::islowVoltageError;
+constexpr std::bitset<3> FaultInjector::isCapLossError;
+constexpr std::bitset<3> FaultInjector::isThermalError;
 
 FaultInjector::FaultInjector(ros::NodeHandle node_handle)
```

Comment:
Minor comment: const ros::NodeHandle& is preferable

Reply:
I don't think you can use const here.

Reply:
Right, it's non-const, but it seems that reference (&) is required to do the right thing, otherwise this function acts on a copy of the node handle.   This is no longer a minor comment :)